### ISSUE TITLE
3rd step of installation guide unclear

### ISSUE THREAD

**Issue by drfinn15** [AnthonMS/my-cards#17]
Hi,

I've tried to install the repository via HACS but am not sure how to proceed in the 3rd step of the installation guide. For me, I understand that I should paste the two lines of code to my HA config, which throws me an error. May you please explain to me what I should do with those lines instead? Thank you!

**Comment by AnthonMS** [comment]
Okay, now I'm gonna give a detailed explanation here and I can link to this from the Readme.

1. You go to HACS
2. You click on Frontend
3. You click on the three dots in the top right corner (See screenshot 1)
4. In the dropdown you click on 'Custom repositories'
5. In the popup asking for URL and category input as following (Or as in screenshot 2)
URL: https://github.com/AnthonMS/my-cards.git
Category: Lovelace
6. Then just click on add and install the card like any other card

![Screenshot 1](https://user-images.githubusercontent.com/23500433/154763939-d3dc00e1-a4b2-4a35-9250-e38c7e23c473.png)
![Screenshot 2](https://user-images.githubusercontent.com/23500433/154764252-cd93a984-aee8-4afc-81a2-bb06bbdc663d.png)

**Comment by bertmelis** [comment]
Adding the repo is clearly explained and straightforward. But your installation steps include a third one:

>Add something in the line of this to the configuration:
> ```
>     - url: /hacsfiles/my-cards/my-cards.js
>       type: module
> ```

Where does this need go?

**Comment by AnthonMS** [comment]
@bertmelis I don't know what to say. You add this card like you add any other card to HA. If you use the frontend for that, then use the frontend. If you use configuration.yaml, then use that. There's nothing different from this card then any other custom card you install from HACS.

But to answer your question. Those two lines go in the configuration file where you normally put the custom card imports. But you most likel don't use the yaml file, so jsut use the frontend like you usually do.

**Comment by bertmelis** [comment]
Oh, yes. It suddenly all makes sense.

**Comment by Thesinofthetin** [comment]
This is gonna sound really stupid, but where is the custom card imports file?

**Comment by wenstarner** [comment]
Hi, I was confused as well. I think the wording in Step 3 can be made easier to understand, because it currently does indeed sound like, with the HACS installation option, that you need to update your config.



**HACS**

1. Add this custom repo to HACS (use this URL: https://github.com/AnthonMS/my-cards.git).
2. Install add-on.
3. Add cards to your dashboard.
    (a) If you edit your dashboard via the front-end/UI, then you can add this card that way (just as you would add any other card).
    (b) If you edit your dashboard using YAML, then use the following:
          `- url: /hacsfiles/my-cards/my-cards.js`
          `  type: module`

**Comment by Perdiga** [comment]
I am having the this isseu when I try to add this repository to HACS. How can I fix it ?
![image](https://github.com/AnthonMS/my-cards/assets/5549410/e20808f2-9ac4-4f2f-96c6-e129ad4ef94e)

**Comment by Thesinofthetin** [comment]
Are you adding it the Front End repo?  It won't work if you're on the integrations repo menu.

**Comment by meetbryce** [comment]
Where exactly am I supposed to include this snippet?

```
- url: /hacsfiles/my-cards/my-cards.js
  type: module
```

**Comment by Travion1976** [comment]
I had the same issue  - entered the URL , that worked fine , but I couldn't select Lovelace. It wouldn't let me.
Restarted HA did the trick. 
Then I was able to add it

**Comment by spyle2** [comment]
Im not seeing any Frontend or tabs in the HACS menu as of 2024/09/21

**Comment by SwizzB** [comment]
Same issue here repo not compliant

**Comment by Kijan** [comment]
You have to choose Dashboard as Typ and then it works

**Comment by SwizzB** [comment]
@Kijan thanks, I did it last time but without success, and I tried today and it worked perfectly !

**Comment by Pierre-Louis1** [comment]
Hi, I am completely new to HA and cannot work out how to install this on the latest version of HA (See image attached) could someone please explain what I need to do? Thanks in advance.
![Screenshot 2024-10-20 215144](https://github.com/user-attachments/assets/33922ad5-9847-4674-a9c2-8dd83129bc9d)

**Comment by AnthonMS** [comment]
> Hi, I am completely new to HA and cannot work out how to install this on the latest version of HA 

Do you have HACS installed in your Home Assistant? If not, you need to install it. ([Install guide](https://www.youtube.com/watch?v=XOnmJ5LPnIw))

And when that is installed you follow the instructions in the comments further up in this issue.

**Comment by Pierre-Louis1** [comment]
Yes I have HACS installed i have been able to look up different cards and download them but when I type in "my-cards" nothing shows up.
![Screenshot 2024-10-20 220612](https://github.com/user-attachments/assets/d9bf7568-8a90-416e-a960-f32e8c5e9613)

**Comment by AnthonMS** [comment]
> Yes I have HACS installed i have been able to look up different cards and download them but when I type in "my-cards" nothing shows up.

You have to add this repository as a custom repo in HACS. I haven't integrated it into default HACS repo.

1. Click the three dots in your screenshot and click "Custom repositories"
2. Select Category "Lovelace"
3. Paste this URL into the Repository https://github.com/AnthonMS/my-cards.git
4. Click add

Then you will be able to find the card in HACS. And then you add it like you would with any other card. I haven't done that in years, so can't really remember that on the top of my head.

**Comment by Pierre-Louis1** [comment]
Thank you so much for the help got it working.

**Comment by Medic97** [comment]
getting this error trying to add to HACS
<Template AnthonMS/my-cards> Repository structure for v1.0.6 is not compliant

**Comment by vtamm** [comment]
> getting this error trying to add to HACS <Template AnthonMS/my-cards> Repository structure for v1.0.6 is not compliant

@Medic97 set the Type to Dashboard when adding the custom repository.

**Comment by captain-ted** [comment]
> https://github.com/AnthonMS/my-cards.git

Thanks for the explanation but there is no option for 'Lovelace' in the category - just:
Dashboard
Theme
Integration
Template 

Which should be used? Theme didn't work.

**Comment by Glennart76** [comment]
> 5\. In the popup asking for URL and category input as following (Or as in screenshot 2)
>     URL: https://github.com/AnthonMS/my-cards.git
>     Category: Lovelace

This step changed - Instead of "Lovelace" choose "Dashboard"

**Comment by Makker-Heineken** [comment]
I am getting this error when trying to add the addon: GitHub returned 503 for https://api.github.com/repos/anthonms/my-cards
<img width="547" height="322" alt="Image" src="https://github.com/user-attachments/assets/f3998568-9b60-432b-a694-850b5247a763" />
